src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -85,6 +84,7 @@ function isSchemaProperty(context: SdkContext, property: ModelProperty) {
     return !(headerInfo || queryInfo || pathInfo || statusCodeInfo);
 }
 
+/* eslint-disable @typescript-eslint/no-explicit-any -- A special recursion */
```

Comment:
if this rule has been disabled, we should remove this line, same to others

Reply:
removed